Dryad Digital Repository.js

```diff
@@ -0,0 +1,715 @@
+{
+	"translatorID": "7a81d945-7d9c-4f8c-bd7b-4226c1cab40e",
+	"label": "Dryad Digital Repository",
+	"creator": "Nathan Day",
+	"target": "^https?://(?:www\\.)?datadryad\\.org/",
+	"minVersion": "1.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2015-05-01 19:04:28"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2014 Dryad Digital Repository
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	// Dryad search page
+	if (url.match('^https?://(?:www\\.)?datadryad\\.org/discover')) {
+		return 'multiple';
+	} else {
+		var result = ZU.xpathText(doc,'//meta[@name="DC.type"][1]/@content');
+		// Dryad data package
+		if (result === 'Article') {
+			return 'journalArticle';
+		// Dryad data file
+		} else if (result === 'Dataset') {
+			//return 'dataset';
+			return 'journalArticle';
+		}
+	}
+	return false;
+}
+
+var packageXpath    = '//meta[@name="DCTERMS.isPartOf"]/@content';
+var multiplesXpath  = '//li[contains(@class,"ds-artifact-item")]/div/a/@href';
```

Comment:
don't include the `/@href` part. Instead, see my comment below where you handle multiples.